packages/react-components/react-progress/src/stories/Progress/ProgressDefault.stories.tsx

```diff
@@ -1,4 +1,28 @@
 import * as React from 'react';
-import { Progress, ProgressProps } from '@fluentui/react-progress';
+import { Progress } from '@fluentui/react-progress';
 
-export const Default = (props: Partial<ProgressProps>) => <Progress {...props} />;
+const intervalDelay = 100;
+const intervalIncrement = 0.01;
+
+export const Default = () => {
+  const [percentComplete, setPercentComplete] = React.useState(0);
+
+  React.useEffect(() => {
+    const id = setInterval(() => {
+      setPercentComplete((intervalIncrement + percentComplete) % 1);
+    }, intervalDelay);
+    return () => {
+      clearInterval(id);
+    };
+  });
```

Comment:
Ditto to above, we shouldn't "animate" the progress by default in the stories since it's not a feature of Progress. You could just set percentComplete={30} or something as a demo.